docs_src/design/adr/core/0021-Device-Profile-Changes.md

```diff
@@ -0,0 +1,64 @@
+# Changes to Device Profiles
+
+## Status
+** Proposed **
+(as of 1/20/22)
+
+Please see a [prior PR](https://github.com/edgexfoundry/edgex-docs/pull/605) on this topic that detailed much of the debate and context on this issue.  For clarity and simplicity, that PR was closed in favor of this simpler ADR.
+
+## Context
+While the device profile has always been the way to describe a device/sensor and template its communications to the rest of the EdgeX platform, over the course of EdgeX evolution there have been changes in what could change in a profile (often based on its associations to other EdgeX objects).  This document is meant to address the issue of change surrounding device profiles in EdgeX going forward – specifically when can a device profile (or its sub-elements such as device resources) be added, modified or removed.
+
+### Summary of Device Profile Rules
+These rules will be implemented in core metadata on device profile API calls.
+
+- A device profile can be added anytime
+- Device resources or device commands can be added to a device profile anytime
+- Attributes and properties can be added to a device profile anytime
+- A device profile can be modified when the device profile is not associated to a device or provision watcher
```

Comment:
A device profile cannot be removed when the device profile is not associated to a provision watcher, but I thought modification is fine.

Reply:
@cloudxxx8 I **think** what you meant was that _while a device profile cannot be removed if associated to a provision watcher, it can be modified_. Is that correct?

Reply:
```suggestion
- A device profile can be removed or modified when the device profile is not associated to a device or provision watcher
```

Reply:
yes, you are correct. sorry, my statement was wrong...

Reply:
altered per above.